api/signup.go

```diff
@@ -104,9 +104,7 @@ func (a *API) Signup(w http.ResponseWriter, r *http.Request) error {
 				return UserExistsError
 			}
 
-			if err := user.UpdateUserMetaData(tx, params.Data); err != nil {
-				return internalServerError("Database error updating user").WithInternalError(err)
-			}
```

Comment:
Hmm but this would mean that the user metadata submitted on the first signup will always be the one used. I guess this is fine since it will mimic the same behaviour for passwords too where subsequent passwords set on signup won't overwrite the first password used to signup.